dev/extra-ml-requirements.txt

```diff
@@ -22,6 +22,8 @@ tensorflow==1.15.2
 torch
 torchvision
 pytorch_lightning>=1.0.2
+transformers
+torchtext
```

Comment:
```suggestion
```

Can you remove `transformers` and `torchtext` because the`mlflow.pytorch` integration doesn't require these two packages.

Reply:
They are required for running the Bert example (using `python`). Is there any specific requirements yaml file we maintain for examples ?

Reply:
We no longer need them once we've addressed https://github.com/mlflow/mlflow/pull/3650/files#r532381100